### PR TITLE
releases: add deprecated note in title

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add deprecated note in titles.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: releases/3.1.0-ga.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: TiDB 3.1 GA Release Notes
+title: TiDB 3.1 GA Release Notes（已废弃）
```
Comment (scsldb) [inline releases/3.1.0-ga.md:2]: I suggest not marked 'deprecated'

File: releases/3.0.10.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
 
 发版日期：2020 年 2 月 20 日
 
 TiDB 版本：3.0.10
 
 TiDB Ansible 版本：3.0.10
 
+> **警告：**
+>
+> 此版本已废弃。
```
Comment (lilin90) [inline releases/3.0.10.md:16]: @scsldb 

- How about suggesting users choose a specific version? So users know which version to use.
- Please help confirm whether it's appropriate to say 废弃 here. Users might be confused. Would it be better to give some explanations?

File: releases/3.0.9.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: TiDB 3.0.9 Release Notes
-category: Releases
+category: Releasess
```
Comment (yikeke) [inline releases/3.0.9.md:3]:
```suggestion
category: Releases
```

File: releases/4.0.0-rc.md [hunk 4]
```diff
@@ -11,6 +11,10 @@ TiDB 版本：4.0.0-rc
 
 TiUP 版本：0.0.3
 
+> **警告：**
```
Comment (yikeke) [inline releases/4.0.0-rc.md:14]: 还有一个问题需要注意下：如果 4.0.0-rc.md 是废弃这种严重级别的话，我建议改一下现有的一些文档中关于使用/升级至 4.0.0-rc.md 的表述，比如 TiUP 有好几篇文档都是用的部署 v4.0.0-rc.md 集群的例子，我担心这样会误导用户去下载 v4.0.0-rc.md 版本。段老板觉得呢？ @scsldb

**Comment by scsldb** [comment]
LGTM

File: releases/4.0.0-rc.md (hunk 4 above)
Comment (lilin90) [reply releases/4.0.0-rc.md:14]: @yikeke 赞同
@TomShawn 来跟进请相关同学处理下吧~ Thanks!

**Comment by TomShawn** [comment]
> @TomShawn 来跟进请相关同学处理下吧~ Thanks!

@lilin90 好的

**Pull request merged**